src/Microsoft.OData.Core/Batch/ODataBatchOperationHeaders.cs

```diff
@@ -75,7 +71,7 @@ public void Add(string key, string value)
         /// <remarks>This method will only try to match the key using case-sensitive comparison.</remarks>
         public bool ContainsKeyOrdinal(string key)

```

Comment:
The `xml` documentation of this method still indicate `...case-sensitive comparison..`.  Should be changed.